tests/Maker/FunctionalTest.php

```diff
@@ -35,7 +36,7 @@ public function testWiring()
         foreach ($finder as $file) {
             $class = 'Symfony\Bundle\MakerBundle\Maker\\'.$file->getBasename('.php');
 
-            if (AbstractMaker::class === $class) {
+            if (AbstractMaker::class === $class || AbstractDockerMaker::class === $class) {
```

Comment:
I guess we could use reflection to check if the class abstract - I don't care too much either way

Reply:
I concur, but would reflection add more overhead?

Reply:
I think the overhead of Reflection has largely (or completely) been disproven. The real overhead, I believe, would be that using Reflection would cause you to autoload these classes. So, it would be super minor - and not important in a test anyways.

Reply:
done